spark-doris-connector/src/main/java/org/apache/doris/spark/rest/RestService.java

```diff
@@ -560,6 +582,42 @@ static List<BackendRow> parseBackend(String response, Logger logger) throws Dori
         return backendRows;
     }
 
+    /**
+     * get Doris BE node.
+     * @param logger slf4j logger
+     * @return the Doris BE node
+     * @throws IllegalArgumentException BE nodes is illegal
+     */
+    public static List<BackendV2.BackendRowV2> getBeBackendRows(SparkSettings sparkSettings,  Logger logger) throws DorisException {
```

Comment:
Backend means be. Maybe we can change the function name?

Reply:
Ok, let me think about what I should call it